pkg/k8s/configmap_test.go

```diff
@@ -0,0 +1,94 @@
+package k8s
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apiv1 "github.com/percona/percona-server-mysql-operator/api/v1"
+	"github.com/percona/percona-server-mysql-operator/pkg/version"
+)
+
+func TestConfigMap(t *testing.T) {
+	cluster := &apiv1.PerconaServerMySQL{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cluster1",
+			Namespace: "configmap-ns",
+		},
+		Spec: apiv1.PerconaServerMySQLSpec{
+			CRVersion: version.Version(),
+		},
+	}
+
+	const name = "configmap-name"
+	const filename = "some-file"
+	const data = "some-data"
+	const component = "some-component"
+
+	t.Run("basic", func(t *testing.T) {
+		cr := cluster.DeepCopy()
+
+		cfg := ConfigMap(cr, name, filename, data, component)
+		assert.Equal(t, name, cfg.Name, name)
```

Comment:
Here the expected message is `name`, is this the intention. I think we should just remove it.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/1101/commits/2b0cad0062a2cd83556ee775961ce4633a8c196d